docs/docs/integrations/libraries/dbt/dbt-migration-components.md

```diff
@@ -0,0 +1,112 @@
+---
+title: Migrating to dbt components
+sidebar_label: Migrating to dbt components
+description: How to migrate a Dagster dbt project from Pythonic to components.
+sidebar_position: 200
+---
+
+Dagster supports two main ways to integrate with dbt, [components](/integrations/libraries/dbt) and the creating assets with the [Pythonic API](/integrations/libraries/dbt/dbt-pythonic). If you have already built your Dagster and dbt project in the Pythonic way, you can migrate to using components and get the same result.
+
+## 1. Scaffold the dbt component
+
+The first step is to [scaffold the dbt component](/integrations/libraries/dbt#3-scaffold-a-dbt-component-definition). This will generate the `defs.yaml` configuration file with a path to your dbt project:
```

Comment:
```suggestion
The first step is to [scaffold a dbt component definition](/integrations/libraries/dbt#3-scaffold-a-dbt-component-definition). This will generate the `defs.yaml` configuration file with a path to your dbt project:
```

Reply:
Do you need to migrate your project with [this guide](https://docs.dagster.io/guides/build/projects/moving-to-components/migrating-project) first?